.github/dependabot.yml

```diff
@@ -1,83 +1,14 @@
 version: 2
 updates:
-  - package-ecosystem: "bundler"
-    directory: "/common"
-    schedule:
-      interval: "weekly"
   - package-ecosystem: "bundler"
     directory: "/updater"
     schedule:
       interval: "weekly"
 
-  # Watch the per-ecosystem gemspecs
-  - package-ecosystem: "bundler"
-    directory: "/bundler"
-    schedule:
-      interval: "weekly"
-  - package-ecosystem: "bundler"
-    directory: "/cargo"
-    schedule:
-      interval: "weekly"
-  - package-ecosystem: "bundler"
-    directory: "/composer"
-    schedule:
-      interval: "weekly"
-  - package-ecosystem: "bundler"
-    directory: "/docker"
-    schedule:
-      interval: "weekly"
-  - package-ecosystem: "bundler"
-    directory: "/elm"
-    schedule:
-      interval: "weekly"
-  - package-ecosystem: "bundler"
-    directory: "/git_submodules"
-    schedule:
-      interval: "weekly"
-  - package-ecosystem: "bundler"
-    directory: "/github_actions"
-    schedule:
-      interval: "weekly"
-  - package-ecosystem: "bundler"
-    directory: "/go_modules"
-    schedule:
-      interval: "weekly"
-  - package-ecosystem: "bundler"
-    directory: "/gradle"
-    schedule:
-      interval: "weekly"
-  - package-ecosystem: "bundler"
-    directory: "/hex"
-    schedule:
-      interval: "weekly"
-  - package-ecosystem: "bundler"
-    directory: "/maven"
-    schedule:
-      interval: "weekly"
-  - package-ecosystem: "bundler"
-    directory: "/npm_and_yarn"
-    schedule:
-      interval: "weekly"
-  - package-ecosystem: "bundler"
-    directory: "/nuget"
-    schedule:
-      interval: "weekly"
   - package-ecosystem: "bundler"
```

Comment:
Maybe add a code comment that we don't need to watch the per-ecosystem bundlers because they roll up to the `"/omnibus"` watcher... because otherwise I could see someone getting confused.

Reply:
I will 👍.

Reply:
Done!